### DISCUSSION TITLE
Not able to run sample project

### DISCUSSION THREAD

**Discussion by vishakha-618** [DanWBR/dwsim#576] [General]
Hii Daniel

There is a inbuilt sample project with the name of "Petroleum distillation".

I just try to run. It is showing error.

### **i did not modify it. I just opened it directly from the sample option and tried to run it.**

![Screenshot 2024-01-16 185735](https://github.com/DanWBR/dwsim/assets/156805573/e7262aad-02a0-484b-8e46-03fd6861a9be)

**Comment by vishakha-618** [comment]
done

**Comment by DanWBR** [comment]
@vishakha-618 this is the included sample, not the one that you took the screenshot from. you are not using an official version of DWSIM then.

![image](https://github.com/DanWBR/dwsim/assets/2669419/fc3c7b54-497f-4d3c-878f-67b4114046de)
